lnwallet/channel_test.go

```diff
@@ -555,6 +556,38 @@ func TestSimpleAddSettleWorkflow(t *testing.T) {
 	}
 }
 
+func TestCheckSize(t *testing.T) {
+	// Create a test channel which will be used for the duration of this
+	// unittest. The channel will be funded evenly with Alice having 5 BTC,
+	// and Bob having 5 BTC.
+	aliceChannel, bobChannel, cleanUp, err := createTestChannels(3)
+	if err != nil {
+		t.Fatalf("unable to create test channels: %v", err)
+	}
+	defer cleanUp()
+
+	b, w:= EstimateCommitmentTxCost(bobChannel)
+
+	// First we test the channel initiator requesting a cooperative close.
+	summary, err := aliceChannel.ForceClose()
+	if err != nil {
+		t.Fatalf("unable to initiate alice force close: %v", err)
+	}
+
+	closeTx := summary.CloseTx
+
+	t.Log("[Estimated] Base size:", b)
+	t.Log("[Estimated] Witness size:", w)
+	t.Log("[Estimated] Cost:", WitnessFactor * b + w)
+
+	stripped := closeTx.SerializeSizeStripped()
+	total := closeTx.SerializeSize()
+
+	t.Log("[Actual] Base size:", stripped)
+	t.Log("[Actual] Witness size:", total - stripped)
+	t.Log("[Actual] Cost:", WitnessFactor * stripped + total - stripped)
```

Comment:
There's a function within the `blockchain` package for directly computing cost: `blockchain.GetTransactionCost`.